amm/contracts/router/lib.rs

```diff
@@ -62,6 +116,18 @@ pub mod router {
             }
         }
 
+        #[inline]
+        fn get_fee(&self, token_0: AccountId, token_1: AccountId) -> Result<u8, RouterError> {
```

Comment:
`get_pair` and `get_fee` can be combined for clarity (a function that returns both), if we want both functions `get_pair` and `get_fee` then they can be implemented on top

Reply:
good idea, will do